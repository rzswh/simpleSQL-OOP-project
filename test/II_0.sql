CREATE DATABASE ldhuhp;
CREATE DATABASE dzj;
CREATE DATABASE vghyv;
USE ldhuhp;
CREATE TABLE ldhuhp(ppbkoqnvgm INT, gregdodxcv INT, keszrwtlfu INT, lqhnzjnnnbz INT NOT NULL, PRIMARY KEY(lqhnzjnnnbz));
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (188, NULL, 154, 3);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (173, 147, NULL, 10);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (192, 158, 165, 7);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (159, 182, NULL, 8);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (138, 161, NULL, 5);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (151, 149, 171, 2);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (NULL, NULL, 180, 4);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (183, 194, 194, 6);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (160, 185, NULL, 9);
INSERT INTO ldhuhp(ppbkoqnvgm, gregdodxcv, keszrwtlfu, lqhnzjnnnbz) VALUES (NULL, 152, 131, 1);
SELECT COUNT(ppbkoqnvgm) from ldhuhp;
SELECT COUNT(gregdodxcv) from ldhuhp;
SELECT COUNT(lqhnzjnnnbz) from ldhuhp;
SELECT COUNT(gregdodxcv) from ldhuhp;
DROP DATABASE ldhuhp;
DROP DATABASE dzj;
DROP DATABASE vghyv;
