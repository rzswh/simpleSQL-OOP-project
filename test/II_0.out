COUNT(ppbkoqnvgm)
8
COUNT(gregdodxcv)
8
COUNT(lqhnzjnnnbz)
10
COUNT(gregdodxcv)
8
